User/SourceControl/Git/Utils.cls

```diff
@@ -1297,14 +1340,14 @@ ClassMethod TestDialogue(pMenuCommand, pCommandValue = "")
 	}
 }
 
-ClassMethod LoadSettings(pRepoBaseFolder, pUserName, pPersonName, pPersonEmail, pTeamBranch, pRepoURL = "", pRepoProtocol = "", pReleaseBranchBase = "", pInstallNamespace = "") As %Status
+ClassMethod LoadSettings(pRepoBaseFolder="", pUserName="", pPersonName="", pPersonEmail="", pTeamBranch="", pRepoURL = "", pRepoProtocol = "", pReleaseBranchBase = "", pInstallNamespace = "") As %Status
```

Comment:
we can now update any individual setting from command line.  nice for testing/debugging things